api/v1alpha1/helmchartproxy_types.go

```diff
@@ -65,6 +65,14 @@ type HelmChartProxySpec struct {
 	// include options such as wait, skipCRDs, timeout, waitForJobs, etc.
 	// +optional
 	Options *HelmOptions `json:"options,omitempty"`
+
+	// CredentialsSecretRef is a reference to a Secret containing the OCI credentials. If it is not specified, no Secret will be used.
+	// +optional
+	CredentialsSecretRef *corev1.SecretReference `json:"credentialsSecretRef,omitempty"`
+
+	// EnableClientCache is a flag to enable helm client cache. If it is not specified, it will be set to true.
+	// +optional
+	EnableClientCache *bool `json:"enableClientCache,omitempty"`
```

Comment:
It could be under Helm option, to be honest in Helm it's not even really exposed as a flag - though it looks like it would be more of a global flag rather than a Helm install specific flag (which is what the HelmOptions currently reflect) though I agree it would make sense to move it to the options